src/CmsBundle/Resources/config/services.yml

```diff
@@ -29,6 +29,12 @@ services:
         tags:
             - { name: doctrine.event_subscriber }
 
+    opifer.contacts_manager:
+        class: Opifer\MailingListBundle\Block\Service\ContactsManager
+        arguments: ['@doctrine.orm.default_entity_manager', '@service_container', '@logger', '%consumer_key%', '%consumer_secret%']
```

Comment:
This requires the user of this bundle to define the parameters. Can you please create Bundle configuration for it?